BUILD_COMMANDLINE.md

```diff
@@ -34,7 +34,7 @@
      OpenJDK 64-Bit Server VM Zulu17.32+13-CA (build 17.0.2+8-LTS, mixed mode, sharing)
      ```
 
-     If the version number indicated is lower than `8.0.0` or if the command returns an empty string, please update or install Java according to an installation instruction matching your operating system (to be found on the internet).
+     If the version number indicated is lower than `8.0.0` or if the command returns an empty string, please update or install Java according to an installation instruction matching your operating system (to be found on the internet). For MacOS users, **make sure to install the Java Development Kit (JDK).**
```

Comment:
It's possible to install this with Homebrew too. Maybe add a comment also here?

Reply:
Also applies for Ant

Reply:
I see two options: either add recommended installation methods for MS Windows and Linux systems or remove the macOS specifics.

Reply:
```suggestion
If the version number indicated is lower than `8.0.0` or if the command returns an empty string, please update or install Java according to an installation instruction matching your operating system (to be found on the internet). The Java Development Kit we use in our Docker Container is Eclipse Temurin™, which is easy to [install](https://adoptium.net/de/installation/) on Linux, macOS or Windows.
```

Reply:
I cannot directly commit this suggestion, @bwbohl, because the file was outdated based on some new commits. But I just added these suggestions in https://github.com/music-encoding/music-encoding/pull/1193/commits/908788912915c3c03deea4bfeca65c9ebabcb7bf.